python/examples/integrations/fastapi/api.py

```diff
@@ -0,0 +1,65 @@
+
+import joblib 
+from fastapi import FastAPI
+from pydantic import BaseModel
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+
+import os
+import whylogs as why
+
+# Set class names for prediction
+CLASS_NAMES = ["setosa", "versicolor", "virginica"]
+
+# Load model
+model = joblib.load('knn_model.pkl')
+
+# Set WhyLabs variables
+os.environ["WHYLABS_DEFAULT_ORG_ID"] = "org-qdcnk5"
+os.environ["WHYLABS_DEFAULT_DATASET_ID"] = "model-77"
+os.environ["WHYLABS_API_KEY"] = "NzhivaYyRM.j1di4X4maxSYkf85fwMOaxRusOh96fQSKp1ifRVCjTYsB7w4z4vpG"
```

Comment:
you should not commit api keys and secrets here

Reply:
thanks for catching, I added back in for testing and forgot to remove!